src/components/MAPI/clusters/CreateCluster/__tests__/schemaUtils.ts

```diff
@@ -1,6 +1,8 @@
+import { RJSFSchema } from '@rjsf/utils';
 import cleanDeep, { CleanOptions } from 'clean-deep';
+import testSchema from 'UI/JSONSchemaForm/test.schema.json';
```

Comment:
Maybe we can mock the test schema somehow? I think test schema will be modified often, and it will break the tests unexpectedly.

Reply:
Sounds like a good idea, I'll create a separate mock schema for tests 👍